core/primitives/src/state.rs

```diff
@@ -113,11 +113,11 @@ impl FlatStateValue {
 
 /// Value to insert into the trie.
```

Comment:
nit: we should also say it can be used to either insert new value or update existent

It's not obvious at first glance